### DISCUSSION TITLE
JournalStorage in optuna-dashboard

### DISCUSSION THREAD

**Discussion by matteo-bastico** [optuna/optuna#4361] [Q&A] [answered]
Hello, 

I am using a Slurm based cluster in which it is not possible to set up a database server and I think that, in this case, the newly introduced JournalFileStorage is the best option to perform distributed optimization. 

I tried to open the log file using the optuna-dashboard and I had the following error

`sqlalchemy.exc.ArgumentError: Could not parse SQLAlchemy URL from string 'journal.log' `

Is it possible somehow to visualize the results stored in a JournalFile using the dashboard? 

Thank you!

**Comment by nzw0301** [comment, marked as answer]
According to [dashboard implementation](https://github.com/optuna/optuna-dashboard/blob/81451c6fd42507dda00bb9255506d8a040453273/optuna_dashboard/_app.py#L405-L415), we cannot use optuna-dashboard with Journal storage.  One workaround is to copy your study from journal storage to an RDB storage, such as SQLite for optuna dashboard. 

I also suggest creating a feature request issue at https://github.com/optuna/optuna-dashboard/issues such that optuna dashboard maintainer might give better ways or implement such a feature.
